chapter2.Rmd

```diff
@@ -201,7 +201,7 @@ success_msg("Well done!")
 --- type:NormalExercise lang:r xp:100 skills:1
 ## Naming a vector (2)
 
-If you want to become a good statistician, you have to become lazy. (If you are already lazy, chances are high you are one of those exceptional, natural-born statistical talents.)
+If you want to become a good statistician, you have to become lazy. (If you are already lazy, chances are high you are one of those exceptional, natural-born statistical talents!).
```

Comment:
I agree this should be an exclamation point here! But you can kill the period at the end (I think rule 2a applies here... http://www.grammarbook.com/punctuation/parens.asp)